src/components/IGV/TrackColourPicker/index.tsx

```diff
@@ -28,6 +28,11 @@ function maybeGetAttributeValue(feature, attrPossibleNames: string[]) {
   return null;
 }
 
+function determineNcRnaPresence(feature) {
+  if (!feature || !feature.getAttributeValue) return null;
```

Comment:
Probably makes no difference – either way the object is instantiated – but shouldn't this be more explicitly checking for something like `if (!feature?.type) return null;` ?